lib/linux/util.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const childProcess = require('child_process');
+const util = require('util');
+const fs = require('fs');
+
+const execFile = util.promisify(childProcess.execFile);
+
+exports.commandExists = async cmd => {
+	// `which` all commands and expect stdout to return a positive
+	try {
+		let {stdout} = await execFile('which', ['-a', cmd]);
+		stdout = stdout.trim();
+
+		if (!stdout) {
+			return false;
+		}
+
+		return true;
+	} catch (_) {
+		return false;
+	}
+};
+
+exports.grep = (stdout, stringToFind) => {
```

Comment:
```suggestion
exports.hasLines = (string, lineToFind) => {
```

Reply:
I think hasLine without an s is more appropriate, because we are searching just one line